src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers, applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
[REQUIRED] You are expected to use createAsyncThunk from the Redux toolkit. Kindly switch to Redux toolkit